src/ontology/edits/oeo-model.omn

```diff
@@ -3678,10 +3702,15 @@ Individual: OEO_00000287
 Individual: OEO_00000288
 
     Annotations: 
-        rdfs:label "vba"@en
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Visual Basic for Applications (VBA) is a programming language based on Visual Basic and developed by Microsoft for use within the Microsoft Office software package."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000118> "VBA"@en,
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/859
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1869",
+        rdfs:label "Visual Basic for Applications"@en,
+        skos:relatedMatch <http://www.ebi.ac.uk/swo/SWO_0000130>
```

Comment:
SWO has only Visual Basic, but not VBA separately. Visual Basic in SWO points to http://www.ebi.ac.uk/swo/SWO_0000130 and there Visual Basic and VBA are treated as synonyms (at least as far as I understand that page).

VBA is derived from Visual Basic, but there is an important difference: While Visual Basic is a fully-fledged programming language that can be used to program stand-alone applications, VBA is much less powerful and can only be used within Microsoft Office applications. I therefore used intentionally only `skos:relatedMatch` here.